edge/stack/stack.go

```diff
@@ -218,7 +218,7 @@ func (manager *StackManager) next() *edgeStack {
 	return nil
 }
 
-func (manager *StackManager) SetEngineStatus(engineStatus engineType) error {
+func (manager *StackManager) SetEngineStatus(assetsPath string, engineStatus engineType) error {
```

Comment:
Feels weird to me that the function to set the engine status requires the assetspath, since this property is not dynamic and cannot change at runtime, wouldn't it make more sense to inject it when creating the stackManager object?